tests/robustness/multiclient_test/framework/snapshotter.go

```diff
@@ -67,6 +67,15 @@ func (mcs *MultiClientSnapshotter) ConnectOrCreateRepo(repoPath string) error {
 	return err
 }
 
+// SetCacheSizeLimits sets the cache size limits for an existing repository
+// the repository server is connected to.
+func (mcs *MultiClientSnapshotter) SetCacheSizeLimits(cacheArgs string) error {
```

Comment:
What is `cacheArgs`? it seems that an arbitrary flag can be passed.

Reply:
Added restrictions for allowed flags